webapp/portlet/src/main/webapp/vue-apps/attach-image/components/PreviewAttachment.vue

```diff
@@ -0,0 +1,144 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2023 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <div>
+    <v-dialog
+      v-model="dialog"
+      width="84vw"
+      :persistent="false"
+      overlay-opacity="0.9"
+      content-class="uiPopup overflow-y-initial "
+      max-width="80vw">
+      <div class="ignore-vuetify-classes ClearFix preview-attachment-action d-flex justify-end">
+        <a 
+          class="icon-large-size white--text"
+          aria-hidden="true"
+          :href="url" 
+          :download="filename">
```

Comment:
```suggestion
          :href="url" 
          :download="filename"
          class="icon-large-size white--text"
          aria-hidden="true">
```
props order (same for others, please)

Reply:
done